src/common/components/scrollIntoViewWithFocus/ScrollIntoViewWithFocus.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+interface Props {
+  children: React.ReactNode;
+  isFocused: boolean;
+}
+
+const ScrollIntoViewWithFocus: React.FC<Props> = ({ children, isFocused }) => {
+  const selfRef = React.useRef<HTMLDivElement | null>(null);
+
+  React.useEffect(() => {
+    if (isFocused && selfRef.current && selfRef.current.scrollIntoView) {
```

Comment:
Would optional chaining: selfRef.current?.scrollIntoView simplify this if statement?

Reply:
Fixed